.github/workflows/ci-build.yml

```diff
@@ -10,37 +10,43 @@ on:
   schedule:
     - cron: '0 3 * * *' # daily, at 3am
 
+concurrency:
+   group: ci-${{ github.head_ref || github.ref }}
+   cancel-in-progress: true
+
 jobs:
   test:
-    name: Tests
+    name: "Tests"
     runs-on: ubuntu-latest
 
     steps:
```

Comment:
Did we lose testing against multiple node versions here?

Reply:
I forgot to answer this!

There is no need, because the output of this repo is browser-code, so node isn't relevant.